modules/memfault-firmware-sdk/CMakeLists.txt

```diff
@@ -31,6 +31,10 @@ zephyr_library_sources_ifdef(
         CONFIG_MEMFAULT_NCS_BT_METRICS
         memfault_bt_metrics.c)
 
+zephyr_library_sources_ifdef(
+        CONFIG_MEMFAULT_NCS_POST_COREDUMP_ON_NETWORK_CONNECTED
+        memfault_lte_coredump.c)
```

Comment:
cmake indent is 2 spaces and the end bracket just randomly changes above, please open another pr fixing the style in this file

Reply:
Styling aligned with the rest of the file, agree it should be addressed in a separate PR